src/nvim/api/vim.c

```diff
@@ -2451,3 +2452,32 @@ void nvim_del_user_command(String name, Error *err)
 {
   nvim_buf_del_user_command(-1, name, err);
 }
+
+/// Get the command line text.
+///
+/// @param[out] err Error details, if any.
+String nvim_get_cmdline(Error *err)
+    FUNC_API_SINCE(9)
```

Comment:
You will need to bump this (and the one below) to `10` after the release on Friday :)